src/daemon/dlt_daemon_common.c

```diff
@@ -368,6 +442,13 @@ int dlt_daemon_free(DltDaemon *daemon, int verbose)
       free(daemon->app_id_log_level_settings);
     }
 #endif
+#ifdef DLT_TRACE_LOAD_CTRL_ENABLE
+    if (daemon->preconfigured_trace_load_settings != NULL) {
+        free(daemon->preconfigured_trace_load_settings);
+        daemon->preconfigured_trace_load_settings = NULL;
```

Comment:
Is it required to reset preconfigured_trace_load_settings_count to 0 after deallocating.

Reply:
Yes, it is good practice, to make sure we are not using empty trace load settings